package.json

```diff
@@ -2,9 +2,42 @@
   "name": "wonder-stuff",
   "version": "1.0.0",
   "description": "Packages for sharing features across JavaScript-based projects",
-  "main": "index.js",
   "repository": "git@github.com:Khan/wonder-stuff.git",
   "author": "Jeff Yates <jeff@khanacademy.org>",
   "license": "MIT",
-  "private": true
-}
+  "private": true,
+  "workspaces": [
+    "packages/*",
+    "build-settings"
+  ],
+  "devDependencies": {
+    "@babel/core": "^7.15.5",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/preset-env": "^7.15.6",
+    "@babel/preset-flow": "^7.14.5",
+    "@khanacademy/eslint-config": "^0.1.0",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^27.2.4",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-flowtype": "^6.1.0",
+    "eslint-plugin-import": "^2.24.2",
+    "eslint-plugin-jest": "^24.5.0",
+    "eslint-plugin-monorepo": "^0.3.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-promise": "^5.1.0",
+    "flow-bin": "^0.161.0",
+    "jest": "^27.2.4",
+    "jest-extended": "^0.11.5",
+    "prettier": "^2.4.1"
+  },
+  "scripts": {
+    "clean": "rm -rf packages/wonder-stuff-*/dist && rm -rf packages/wonder-stuff-*/node_modules",
+    "coverage": "yarn run jest --coverage",
+    "flow-ci": "flow check --max-workers 0",
+    "jest": "jest --config config/jest/test.config.js",
+    "lint": "eslint --config .eslintrc.js packages",
```

Comment:
Similar, eslint should recognize the .eslintrc.js without the `--config` option.